spec/system/task_spec.rb

```diff
@@ -1,36 +1,36 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) { create(:project) }
+
   describe 'Task一覧' do
+    let(:task) { create(:task, project: project) }
```

Comment:
`association`を定義したことで`project`の記載は必要なくなりますね。
```suggestion
    let(:task) { create(:task) }
```

Reply:
`0bb3feb` で修正